laterpay/application/Core/Bootstrap.php

```diff
@@ -423,6 +426,7 @@ public function activate() {
 
         // register the 'refresh dashboard' cron job
         wp_schedule_event( time(), 'hourly', 'laterpay_refresh_dashboard_data' );
+        wp_schedule_event( time(), 'daily', 'laterpay_delete_old_post_views' );
```

Comment:
It's not helpful to put this under the 'refresh dashboard' cron job with the same comment.
The comment should probably be something like "// register cron jobs".